scipy/sparse/bsr.py

```diff
@@ -13,7 +13,8 @@
 from .data import _data_matrix, _minmax_mixin
 from .compressed import _cs_matrix
 from .base import isspmatrix, _formats, spmatrix
-from .sputils import isshape, getdtype, to_native, upcast, get_index_dtype
+from .sputils import isshape, getdtype, to_native, upcast, get_index_dtype, \
+    check_shape
```

Comment:
Use parentheses in favor of line continuations.

Reply:
I'll fix it.